kafka-broker/kafka

```diff
@@ -17,6 +17,8 @@
 # Short-Description: Starts the Kafka Broker
 ### END INIT INFO
 
+# Init file for acm certificate generation
+/usr/bin/acm-pca-cert-generator
```

Comment:
This will need a bunch of command line flags passing in (see https://github.com/dwp/acm-pca-cert-generator/#running).  One thing to be aware of is that the `--trusted-certs` flag needs an s3 URL passing in, and the bucket name will obviously be environment dependent.  So you're going to have to somehow pass that in or otherwise make it available to this script.  Maybe some user_data to set an env var that you can source from in here? I honestly don't have that much of an idea of what will and won't work here.

Reply:
moved to userdata